### ISSUE TITLE
Errors since a few days

### ISSUE THREAD

**Issue by dxrkii** [NightfallGT/Twitch-Follow-Bot#8]
Hello everyone,
I have got a problem since a few days which leaves me unable to use the bot.

_[{"errors":[{"message":"failed integrity check"}],"extensions":{"challenge":{"type":"integrity"},"durationMilliseconds":10,"operationName":"FollowButton_FollowUser","requestID":"01GE2CJGAQ42DWNTFSJDCCV0V8"}}]_

This happens for every OAuth Token thats in there, no new ones work as well.
Thank you in advance!

**Comment by injectionmethod** [comment]
Should of been answered by now but in laymans terms its due to Twitch changing their api
